src/benchmark.py

```diff
@@ -0,0 +1,235 @@
+    # Compress the dataframe by dropping unneccssary information
+    df = df[['package_ndc', 'generic_name', 'brand_name', 'labeler_name']]
+    # Standardlize dashed NDC to CMS 11 digits NDC
+    df.package_ndc = df.package_ndc.apply(convert_ndc)
+    df.generic_name = df.generic_name.str[:100]
+    if pd2db == "yes":
+        pd_to_postgres(df, table_name, mode)
+    else:
+        df_to_sql(df, table_name, mode, new_table, psql, cur, engine)
+    print(datetime.datetime.now(eastern).strftime("%Y-%m-%dT%H:%M:%S.%f")+' Finish Reading NDC and save in table ndcdata')
+
+def merge_table():
+    query = """
+        SELECT
+            sdudtest.ndc,
+            sdudtest.tot_reimbursed,
+            ndctest.generic_name
+        INTO
+            sdud_cleaned
+        FROM
+            sdudtest
+        LEFT JOIN ndctest ON ndctest.package_ndc = sdudtest.ndc;
+        SELECT * FROM sdud_cleaned LIMIT 10;
+    """
+    cur.execute(query)
+    conn.commit()
+
+def sum_by_state():
+    query = """
+        SELECT SUM(tot_reimbursed), generic_name
+        FROM sdud_cleaned
+        GROUP BY generic_name;
+    """
+    cur.execute(query)
+    conn.commit()
+
+if __name__ == "__main__":
+    # Disable `SettingWithCopyWarning`
+    pd.options.mode.chained_assignment = None
+    eastern = pytz.timezone('US/Eastern')
+    new_table = 0
+    chunk_size = int(sys.argv[1])
+    psql = sys.argv[2]
+    psyc = sys.argv[3]
+    sraw = sys.argv[4]
+    pd2db = sys.argv[5]
+    if psql == "psql":
+        print(datetime.datetime.now(eastern).strftime("%Y-%m-%dT%H:%M:%S.%f")+' Connect to PostgreSQL on AWS RDS')
+        user = os.getenv('POSTGRESQL_USER', 'default')
+        pswd = os.getenv('POSTGRESQL_PASSWORD', 'default')
+        host = 'psql-test.csjcz7lmua3t.us-east-1.rds.amazonaws.com'
+        port = '5432'
+        dbname = 'postgres'
+        surl = 'postgresql://'
+    else:
+        print(datetime.datetime.now(eastern).strftime("%Y-%m-%dT%H:%M:%S.%f")+' Connect to AWS Redshift')
+        user = os.getenv('REDSHIFT_USER', 'default')
+        pswd = os.getenv('REDSHIFT_PASSWORD', 'default')
+        host = 'redshift-test.cgcoq5ionbrp.us-east-1.redshift.amazonaws.com'
+        port = '5439'
+        dbname = 'rxtest'
+        surl = 'redshift+psycopg2://'
+        s3 = s3fs.S3FileSystem(anon=False)
+    if psyc == "psycopg2":
+        print("Using Psycopg2")
+        conn = psycopg2.connect(dbname=dbname, user=user, host=host, port=port, password=pswd)
+    else:
+        engine = sa.create_engine(surl+user+':'+pswd+'@'+host+':'+port+'/'+dbname,echo=False)
+        if sraw == "raw":
+            conn = engine.raw_connection()
+        else:
+            con = engine.connect()
+            conn = con.connection
+    cur = conn.cursor()
+    print(datetime.datetime.now(eastern).strftime("%Y-%m-%dT%H:%M:%S.%f")+' Connected')
+    s3_path = 's3n://rxminer/'
+    start0 = time.time()
+    start = time.time()
+    if (psyc == "sqlalchemy" and sraw == "no") :
```

Comment:
ENUMS again! The main reason I am emphasized this is that as your code-base begins to grow, it would be very tough to keep track of all these strings and making changes would not be very trivial